server/index.js

```diff
@@ -19,11 +24,32 @@ app.get('/houses/*', (req, res) => {
           res.send(rows); 
         })
         .then((res) => {
-          console.log(res);
+          //console.log(res);
```

Comment:
`res.send` and `conn.end` are both synchronous operations. Do these operations need their own separate `.then` block?

Reply:
resolved.